terraform/environment/modules/environment/rds.tf

```diff
@@ -124,6 +124,12 @@ resource "aws_db_parameter_group" "postgres13-db-params" {
     value        = "1440"
     apply_method = "immediate"
   }
+
+  parameter {
+    name         = "log_duration"
```

Comment:
Could you please give me a brief description of what effect `log_duration` has? What does "1" signify?

Reply:
Log duration will print the time taken for an SQL query to execute next to the statement. 1 equates to True